automatic/screencloud/README.md

```diff
@@ -0,0 +1,14 @@
+# ScreenCloud [![Build Status](https://travis-ci.org/olav-st/screencloud.svg?branch=master)](https://travis-ci.org/olav-st/screencloud) 
+
+This is the source code for the ScreenCloud client. ScreenCloud is an easy to use screenshot sharing tool consisting of a cross-platform client and a sharing website: [http://screencloud.net/](http://screencloud.net/)
+
+You can also upload to other services, including your own FTP server, Imgur, Dropbox & Ubuntu One through the use of [plugins](https://github.com/olav-st/screencloud-plugins).
```

Comment:
You'll need to replace `&` with `&amp;`, otherwise `choco pack` will fail because the description is just added as-is.